components/table/ReportTable.tsx

```diff
@@ -1,13 +1,41 @@
 import React from "react"
 import { Flex } from "@chakra-ui/layout"
 import { TimesheetEntry } from "@/lib/types/apiTypes"
+import Loading from "../common/Loading"
+
+interface TotalHoursProps {
+    startDate: string
+    endDate: string
+    totalQuantity: number
+}
 
 interface ReportTableProps {
     entries: TimesheetEntry[]
+    startDate: string
+    endDate: string
+}
+
+function TotalHours({
+    startDate,
+    endDate,
+    totalQuantity,
+}: TotalHoursProps): JSX.Element {
+    if (totalQuantity > 0) {
+        return (
+            <p>
+                The total number of hours between {startDate} and {endDate} is{" "}
+                {totalQuantity}
+            </p>
+        )
+    }
```

Comment:
I think we should allow `totalQuantity = 0` in this component, and render something like
```
                <p>
                    No hours between {entries[0].date} and
                    {entries[responseLength - 1].date}
                </p>
```
if `totalQuantity = 0`.